### PR TITLE
Feat/add api to project

### PR DESCRIPTION
**Pull request by Sirmelihy**
types changed according to our API. And create-space endpoint implemented

### CODE REVIEW

File: .env
Comment (Sirmelihy) [inline .env:1]: considering again, that might not be good idea push to..

Comment (fehmisener) [reply .env:1]: Delete or change the name for local env

File: src/components/CreateSpace/steps/SpaceBasics.tsx [hunk 1]
```diff
@@ -33,9 +41,29 @@ const SpaceBasics: React.FC<SpaceBasicsProps> = ({ data, onUpdate, onNext }) =>
...
+        >
+          Creator Nickname
+        </label>
+        <input
+          type="text"
+          id="creatorName"
+          value={data.creatorNickname}
+          onChange={(e) => onUpdate({ creatorNickname: e.target.value })}
+          className="w-full px-4 py-2 rounded-lg border border-gray-300 focus:ring-2 focus:ring-purple-500 focus:border-transparent"
+          placeholder="Boncuky"
```
Comment (fehmisener) [inline src/components/CreateSpace/steps/SpaceBasics.tsx]: Change

**Review by fehmisener — CHANGES_REQUESTED** [review]
Overall OK but remove the dummy comment blocks, if they're not necessary

**Pull request merged**